ai/Algorithm/Graph/Node.py

```diff
@@ -41,17 +42,8 @@ def add_vertex(self, p_vertex):
                 return
         self.vertices.append(p_vertex)
 
-    def remove_vertex(self, p_ending):
-        """
-        Retire un vertex du noeud, spécifié par le numéro de son noeud d'arrivé. Si le noeud courant ne possède pas de
-        vertex vers le noeud spécifé, rien ne se passe.
-        :param p_ending: Un entier positif représentant le numéro du noeud d'arrivé du vertex à retirer.
-        """
-        assert isinstance(p_ending, int)
-        assert 0 <= p_ending
-        for i in range(len(self.vertices)):
-            if self.vertices[i].next_node == p_ending:
-                self.vertices.pop(i)
+    def connect_to(self, dst_node, *, when: Callable[..., bool]):
```

Comment:
Why the `*,` ?

Reply:
It make sure you use when as a keyword argument

Reply:
The more you know